server/api/controllers/faq/create-faq.js

```diff
@@ -24,14 +28,15 @@ module.exports = {
 
   exits: {},
 
-  fn: async function ({ pageTopic, question, answer, isPublished }, exits) {
+  fn: async function ({ pageTopic, question, answer, order, isPublished }, exits) {
     const createdBy = this.req.user.fullName;
 
     try {
       const newFaq = await Faq.create({
```

Comment:
<img width="384" alt="image" src="https://github.com/hmcc-global/hmcchk-web/assets/24607458/edda918d-f55d-4f4f-b983-cb8a93b4b6bf">

thinking we could use a more graceful error msg

Reply:
or maybe even set the page topic field as a drop down

Reply:
oh yea since it's already enum, can be a dropdown list. just put it in /lists.js

Reply:
ok!